ozaria/site/components/interactive/PageInteractive/common/InteractiveTitle.vue

```diff
@@ -30,16 +30,26 @@
 </template>
 
 <style scoped>
+  /* TODO load this with rest of fonts */
+  @import url('https://fonts.googleapis.com/css?family=Work+Sans&display=swap');
```

Comment:
This needs to be loaded with the rest of our fonts.  Took a look and did not find and place where we do that.   Is anyone aware of how this is done now / is there a plan for Ozaria?

Reply:
I'm aware that fonts are imported here (due to requirement of different font import due to infrastructure):
https://github.com/codecombat/codecombat/blob/547cc82f4126a3d80464830606fdb255c5decf91/app/app.js#L7

There may be other places where fonts are imported.

Reply:
Updated thank you.